akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -74,11 +85,11 @@ After being deprecated since 2.5.0, the following have been removed in Akka 2.6.
 * `akka.stream.actor.ActorSubscriber` and `akka.stream.actor.ActorPublisher`
     - Use `GraphStage` instead.
 
-After being deprecated since 2.4.0, the following have been removed in Akka 2.6.
+After being deprecated since 2.4.0, the following have been removed in Akka 2.6.0.
```

Comment:
we used a mix of version styles here. Changed to 3 digits, either 2.6.x or 2.6.0 depending on context